src/arch/xtensa/up/xtos/reset-vector.S

```diff
@@ -86,7 +86,8 @@ _ResetVector:
 /* Apollolake+ have reset vector in ROM */
 #if defined(CONFIG_BAYTRAIL) || defined(CONFIG_CHERRYTRAIL) \
 	|| defined (CONFIG_HASWELL) || defined(CONFIG_BROADWELL) \
-	|| defined(CONFIG_SKYLAKE) || defined(CONFIG_KABYLAKE)
+	|| defined(CONFIG_SKYLAKE) || defined(CONFIG_KABYLAKE) \
+	|| defined(CONFIG_IMX8)
```

Comment:
this is starting to get ugly, maybe we should have an internal define that's set by the different platforms.

Reply:
yep, something in Kconfig automatically selected by platform.